backend/app/settings.py

```diff
@@ -32,6 +33,11 @@ class Settings(BaseSettings):
     # Base URL of the application
     BASE_URL: str = "http://localhost:3000"
 
+    # For creating HTTP Message Signatures
+    PRIVATE_KEY: RsaPrivateKey
+    # Where to load public keys to verify HTTP Message Signatures
+    HTTP_SIG_VERIFY_JWKS_URI: str = "http://127.0.0.1:5000/.well-known/jwks.json"
```

Comment:
I think the README could also explain that this needs to be configured properly if you want to test it out else than locally.

Reply:
I added mention of this to README